tests/tests/test_integration/test_gui/test_gui.py

```diff
@@ -51,6 +53,13 @@ def check_loaded_layers(widget, num_layers):
     assert widget.atlas_layer.name == widget.atlas.atlas_name
 
 
+def check_editable(widget, standard_space=False):
```

Comment:
Recommend renaming to `check_not_editable` to better describe what it's checking